collect_app/src/main/java/org/odk/collect/android/widgets/ImageWidget.java

```diff
@@ -64,6 +62,7 @@ public ImageWidget(Context context, final QuestionDetails prompt) {
         addAnswerView(answerLayout);
     }
 
+
```

Comment:
Please get rid of this extra empty line. Hmm it should break the build, i'm wondering why it didn't happen in this case.